addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -217,8 +217,8 @@ img.o_forum_avatar_big {
         }
     }
 
-    .o_wforum_bio_popover_info .css_editable_mode_hidden > .mt-2 {
-        margin-top: 0!important;
+    .o_wforum_bio_popover_info .css_editable_mode_hidden > div:last-child > .o_forum_tooltip_line {
+        margin-top: -0.5rem; // compensate parent 'mt-2' class
```

Comment:
@stefanorigano What if the mt-2 is changed by a custo or another refactoring... Anyways, should at least be using the `$spacers` map.

Why can't it be a mt-n2 on the xml element ? (don't touch the branch I am taking care of it, just a question to help me ;))

Reply:
Hello @qsm-odoo , the popup's content is rendered using the 'contact' widget...
Considering that it's just a design/layout detail,  I preferred to avoid touching the original html 'cause of possible cascade effects on other modules/themes...
I assume that there is a reason why such a strong class is used there, but visually it doesn't fits forum.

hope it helps